README.md

```diff
@@ -185,6 +212,7 @@ terraform apply
 ```
 Terraform automatically loads the variables from your `terraform.tfvars` variable definition file.
 Installation times may very, but it is expected to take up to 30 min to complete the deployment.
+It is recommended to use AWS `admin` account, or ask your AWS administrator to assign necessary IAM roles and permissions to your user.

```

Comment:
A small note: Once I started to create this script to generate a policy with all necessary permissions: 
https://github.com/dspace-group/simphera-reference-architecture-aws/blob/main/scripts/least_permissions.sh.
The output of the latest run you find here: https://github.com/dspace-group/simphera-reference-architecture-aws/blob/main/policy.json

The idea is to run the script in a CI/CD pipeline with admin permissions. Users of the terraform modules can then create and use an IAM user with the permissions from policy.json.

However, the work is still unfinished. Never tested the policy. It is outdated anyways.

Reply:
OK, I've checked it, but in any case, you need someone with admin level privileges to add those permissions or yo need to be admin yourself.